src/http/Api/setSatusCode.php

```diff
@@ -0,0 +1,7 @@
+<?php
```

Comment:
weird file name for this functionality.
Better use notFoundCode or something like that.

Reply:
Done